configs/jest.json

```diff
@@ -1,14 +1,15 @@
 {
   "index_name": "jest",
   "start_urls": [
-    "https://jestjs.io/docs/"
+    "https://archive.jestjs.io/"
```

Comment:
@slorber I had to remove the `/docs` as it redirects to a 404 the crawl was ending in some cases